install_config/configuring_pipeline_execution.adoc

```diff
@@ -7,26 +7,32 @@
 :experimental:
 :toc: macro
 :toc-title:
+:prewrap!:
 
 toc::[]
 
 
 == Overview
 
-{product-title} utilizes Jenkins to execute continuous development pipelines.  To
-provide a seamless experience for users, the first time a user creates a build
-configuration using the xref:../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[pipeline]
-strategy type, a set of Jenkins related resources will be created within the user's project.
-By default, the system will look for a template named `*jenkins*` in the `*openshift*` namespace and
-instantiate it within the user's project.  {product-title} ships with a default template definition
-which will, upon instantiation, create the following in the user's project:
+{product-title} utilizes Jenkins to execute continuous development pipelines.
+To provide a seamless experience for users,
+the first time a user creates a build configuration using the
+xref:../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[Pipeline]
+strategy type,
+a set of Jenkins related resources will be created within the user's project.
+By default, the system will look for a template named `*jenkins*`
+in the `*openshift*` namespace and instantiate it within the user's project.
+{product-title} ships with a default template definition which will,
```

Comment:
@tnguyen-rh The installer copies everything from https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_examples/files/examples/ (where v1.x is the origin version of the installation) to the first master and then installs those into the openshift namespace. If it's an enterprise install it gets the rhel image streams, if it's origin it gets centos image streams.